lib/tasks/projects.rake

```diff
@@ -180,4 +180,18 @@ namespace :projects do
       end
     end
   end
+
+  desc 'Update Go Base Modules'
+  task :update_go_base_modules, [:version] => :environment do |_task, args|
+    # go through versioned module names and rerun update on them to get their versions
+    # added to the base module Project
+    Project.where(platform: "Go").where("name like ?", "%/v#{args[:version]}").find_in_batches do |projects|
+      projects.each do |project|
+        matches = PackageManager::Go::VERSION_MODULE_REGEX.match(project.name)
+        Project.create(platform: "Go", name: matches[1])
```

Comment:
Wouldn't we want a find_or_create? (Or create_or_find)

Reply:
I stand corrected, this is fine and is non-explody

Reply:
I was banking on `create` silently failing based on the unique indexes in the database and then continuing on afterwards. I'll switch it out to find_or_create_by since that shows the intention a little better.